src/cli/migrate/button-props.ts

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
Should we worry about versioning this migration as well? maybe `v1-to-v2/button-props` (Anything goes, really), but I think for future proofing we can't know for sure where this can be used. Think of it similar to how we usually name db migrations to avoid conflicts

Reply:
This also helps people migrating from previous versions (v1 to v3). Imagine we change the button props again on v3 and override this current codemod (since it has the same name/purpose), people on v1 would be locked out of it

Reply:
I thought about it and decided to keep it simple for now. I've already changed the `button-props` migration to a more specific name (`button-variants`) in a follow-up PR. Initially, I wasn't sure yet how I would group/structure the codemods, that's why I used a generic name.

> Imagine we change the button props again on v3 and override this current codemod (since it has the same name/purpose), people on v1 would be locked out of it

I don't want to have to support upgrades across two major versions. Users should first upgrade to v2, then v3, i.e. run the v2 codemods first, then the v3 ones.

Reply:
I modeled the codemods after the Storybook ones — they don't use versioning either, just [descriptive names](https://github.com/storybookjs/storybook/tree/next/lib/codemod/src/transforms).

Reply:
I've renamed the transform in this PR to be more specific and descriptive.

Reply:
We are on the same page. I meant users not having access to "older" mods with the same purpose. But you already renamed to a more unique mod.